confmap/provider/eecprovider/yaml.go

```diff
@@ -0,0 +1,21 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package eecprovider // import "github.com/Dynatrace/dynatrace-otel-collector/confmap/provider/eecprovider"
```

Comment:
I would maybe move this to a separate package, since it can be used also outside of this provider

Reply:
That's a good callout, but I would prefer to wait until we need it to be used elsewhere. Right now we only plan to have a single Provider, and we also hope that we will be able to remove this Provider in the future in favor of using the upstream HTTP/HTTPS providers.

Reply:
It's a note for the future, I'll leave it up to you